Allow initializing only subset of labels (#179)
spacemeshos/post: cmd/postcli/README.md

```

### Initialize subset
It is possible to initialize only subset of the files. It's useful if one wants to split initialization between many machines.
#### Example - split initialization between 2 machines
Let's assume one wants to initialize 100 units and split the process of initialization to machine A and B.

First let's see how many files it would be:
```bash
./postcli -numUnits 100 -printNumFiles
1600
```

Next, on machine A:
```bash
./postcli -numUnits 100 -id <id> -commitmentAtxId <id> -toFile 799 -datadir ./dataA
```
We get the first half - 800 binary files
```bash
ls -la ./dataA/*.bin | wc -l
800
```

Next, on machine B:
```bash
./postcli -numUnits 100 -id <id> -commitmentAtxId <id> -fromFile 800 -datadir ./dataB
```

> We get the first half - 800 binary files

Reviewer: ```suggestion
We get the second half - 800 binary files
```

```bash
ls -la ./dataB/*.bin | wc -l
800
```

Finally we can combine both sets together. A dummy example to get the feeling. Realisticly it would probably be copying over the network
```bash
cp ./dataA/* ./data/
cp ./dataB/* ./data/
ls -la ./data/*.bin | wc -l
1600
```

**An optional step to select best possible VRF nonce**

> Normally, when `postcli`initializates from the start to the end it will automatically pick the best vrf nonce. The best means pointing to **the label with the smallest value**. It might be important in the future if we allow increasing POS data size, but it is not critical at the moment.

Reviewer: ```suggestion
Normally, when `postcli`initializates from the start to the end it will automatically pick the best VRF nonce. The best means pointing to **the label with the smallest value**. This will avoid a longer initialization time when a node increases their PoST size in the future (not supported yet).
```

> Now, when `postcli` initializes in chunks, each subset will find a valid vrf nonce, which represents the local minimum in the inititalized subset. It's optimal to select the best one (the global minimum), but it's not critical.

Reviewer: ```suggestion
Now, when `postcli` initializes in chunks, each subset will find a valid vrf nonce, which represents the local minimum in the inititalized subset. It is recommended to select the best one (the global minimum).
```

The values of nonces are 128bit, represented as a 16B binary array in big endian. Given two nonces:

> ```
> nonceA = 0000ffda94993723a980bf557509773e
> nonceB = 0000488e171389cce69344d68b66f6b4
> ```
> `nonceB` represents the smaller value.

Reviewer: ````suggestion
```
NonceA = 12345
NonceValueA = 0000ffda94993723a980bf557509773e
NonceB = 98765
NonceValueB = 0000488e171389cce69344d68b66f6b4
```
`NonceB` is the global minimum since its value is smaller than the one of `NonceA`.
````

> The nonce and the label it points to is included in the post_metadata.json. It's currently up to the operator to find the best VRF nonce manually and copy the `Nonce` and `NonceValue` to the postdata_metadata.json on the target machine.

Reviewer: ```suggestion
The nonce (index) and noncevalue (the label) is included in the post_metadata.json. It is up to the operator to find the best VRF nonce manually and copy the `Nonce` and `NonceValue` to the postdata_metadata.json on the target machine.
```
